MealFridge/Views/Home/Index.cshtml

```diff
@@ -43,7 +41,7 @@
                                     <div class="card">
                                         <img src="@Model[i].Image" alt="Recipe image" class="card-img-top" />
                                         <h5 class="card-title">@Model[i].Title</h5>
-                                        <p class="card-text">@Model[i].Summery</p>
```

Comment:
Why'd we remove this?

Reply:
Wasn't showing up anyways, but just like the try-catch code bit above, I'll put it back in a new pull request.